### ISSUE TITLE
Organise an accessibilty audit

### ISSUE THREAD

**Issue by peteryates** [DFE-Digital/register-early-career-teachers#603]
It's good practice for DfE services to be checked by third party accessibility specialists. Usually this takes place during a private beta but as we're replacing an existing service we'll need to find a suitable time. It isn't something we want to do before the dust has settled development wise but equally we don't want to leave it too long after launch.

Adding an issue to the backlog so we don't forget it.

We should hopefully have plenty of accessibility checks in place (#75) by the point we do it and we'll mainly be sticking to the design system, so there shouldn't be any massive fails.
